src/main/java/br/com/digituz/mailer/service/ScheduledService.java

```diff
@@ -0,0 +1,59 @@
+package br.com.digituz.mailer.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author daniel
+ */
+@Component
+public class ScheduledService {
+
+	@Autowired
+	private JavaMailSender javaMailSender;
+
+	@Value("${spring.mail.username}")
+	private String sender;
+
+	@Value("${mailer.replyTo}")
+	private String replyTo;
+
+	private static final String TIME_ZONE = "America/Sao_Paulo";
+	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+	@Scheduled(cron = "* */5 * * * *", zone = TIME_ZONE)
+	public void sendEmails() {
+
```

Comment:
O que tá te faltando, pelo visto, é buscar a lista de emails não enviados. Isso tu faz através do emailRepository. Depois que tu tiver a lista de emails não enviados, tu faz um for each em cima dela enviando um a um.